src/axom/quest/examples/CMakeLists.txt

```diff
@@ -60,22 +60,25 @@ if(ENABLE_MPI AND MFEM_FOUND AND MFEM_USE_MPI
 
         # Non-zero empty-rank probability tests domain underloading case
         set(_emptyRankProbs 0.0 0.5)
```

Comment:
It looks this parameter was removed. We should remove the variable.

Reply:
Thank you.  Will do.